README.md

```diff
@@ -1,5 +1,8 @@
 # zkLLVM Circuit Compiler
 
+[![build-linux](https://github.com/NilFoundation/zkllvm/actions/workflows/build_linux.yml/badge.svg?branch=master)](
```

Comment:
preview: https://github.com/NilFoundation/zkllvm/tree/nickvolynkin/build-in-ci

will become green after merging